LayoutTests/fast/viewport/ios/wide-desktop-viewport-in-enhanced-windowing-mode.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true enhancedWindowingEnabled=true ShouldIgnoreMetaViewport=true ] -->
+<html>
+<head>
+<script src="../../../resources/js-test.js"></script>
+<script src="../../../resources/ui-helper.js"></script>
```

Comment:
(Whoops — the `ui-helper.js` script is not necessary for this test; will remove..)